### PR TITLE
Ctcp 1985

### PR DESCRIPTION
**Pull request by JohnWhitfield**

### CODE REVIEW

File: src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/Arrivals/TransportEquipmentGoodsRefStepDef.scala [hunk 1]
```diff
@@ -68,14 +68,14 @@ class TransportEquipmentGoodsRefStepDef extends BaseStepDef {
...
-  And("""^(?:I )?click the (.*) link on the You have added (.*) transport equipment page""") {
+  And("""^(?:I )?click the (.*) link on the You have added (.*) transport equipment page$""") {
     (sectionLink: String, numberOfTransportEquipment: String) =>
     AddAnotherTransportEquipmentPage
       .loadPage(numberOfTransportEquipment)
       .clickByPartialLinkText(sectionLink)
   }
 
-  And("""^(?:I )?click radio option (.*) on the Are you sure you want to remove transport equipment (.*) page""") {
+  And("""^(?:I )?click radio option (.*) on the Are you sure you want to remove transport equipment (.*) page$""") {
```
Comment (vajarova) [inline src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/Arrivals/TransportEquipmentGoodsRefStepDef.scala:78]: Shall we keep those consistent?
`"""^(?:I )?click radio option (.*) on the 'Are you sure you want to remove transport equipment (.*)\?' page$"""`

File: src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/Arrivals/TransportEquipmentGoodsRefStepDef.scala [hunk 2]
```diff
@@ -45,14 +45,14 @@ class TransportEquipmentGoodsRefStepDef extends BaseStepDef {
...
-  And("""^(?:I )?click the (.*) link on the You have added (.*) goods item numbers? page""") {
+  And("""^(?:I )?click the (.*) link on the You have added (.*) goods item numbers? page$""") {
     (sectionLink: String, numberOfGoodsItems: String) =>
       AddAnotherGoodsItemNumberPage
         .loadPage(numberOfGoodsItems)
         .clickByPartialLinkText(sectionLink)
   }
 
-  And("""^(?:I )?click radio option (.*) on the Are you sure you want to remove goods item number (.*) page""") {
+  And("""^(?:I )?click radio option (.*) on the Are you sure you want to remove goods item number (.*) page$""") {
```
Comment (vajarova) [inline src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/Arrivals/TransportEquipmentGoodsRefStepDef.scala:55]: Shall we keep those consistent?
`"""^(?:I )?click radio option (.*) on the 'Are you sure you want to remove goods item number (.*)\?' page$"""`

File: src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/Arrivals/IncidentStepDef.scala [hunk 3]
```diff
@@ -86,14 +86,14 @@ class IncidentStepDef extends BaseStepDef {
...
-  And("""^(?:I )?click the (.*) link on the You have added (.*) incidents? page""") {
+  And("""^(?:I )?click the (.*) link on the You have added (.*) incidents? page$""") {
     (sectionLink: String, numberOfIncidentsItems: String) =>
       AddAnotherIncidentPage
         .loadPage(numberOfIncidentsItems)
         .clickByPartialLinkText(sectionLink)
   }
 
-  And("""^(?:I )?click radio option (.*) on the Are you sure you want to remove incident (.*) page""") {
+  And("""^(?:I )?click radio option (.*) on the Are you sure you want to remove incident (.*) page$""") {
```
Comment (vajarova) [inline src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/Arrivals/IncidentStepDef.scala:96]: Keep the consistency?
`"""^(?:I )?click radio option (.*) on the 'Are you sure you want to remove incident (.*)\?' page$"""`

Adding '' and \?

File: src/test/resources/features/DepartureFE/End-to-end-XI.feature [hunk 4]
```diff
@@ -127,15 +127,15 @@ Feature: End to end journey for Northern Ireland office of departure
...
+    And I submit on the 'Transport details - Modes and means of transport Check your answers' page
 
         ## Supply chain actor
     When I choose radio option No on the 'Do you want to add a supply chain actor?' page
 
         ## Authorisation
     And I choose radio option Yes on the 'Do you want to add an authorisation' page
-    And I choose radio option TRD on the 'Which type of authorisation do you want to add' page
-    And I enter TRD123 reference number on the 'What’s the reference number for the TRD authorisation?' page
+    And I choose radio option SSE on the 'Which type of authorisation do you want to add' page
```
Comment (vajarova) [inline src/test/resources/features/DepartureFE/End-to-end-XI.feature:137]: Is there a reason we changed the TRD to SSE?

**Review by vajarova — CHANGES_REQUESTED** [review]

**Review by vajarova — DISMISSED** [review]

**Review by Gosifo — APPROVED** [review]

**Pull request merged**